src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+.conteiner {
+  height: 100px;
+  width: 100%;
+}
+
+.c1 {
```

Comment:
will be better to add more descriptive classes

Reply:
fixed